scripts/eq_translations_check.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import requests
+import sys
+
+import eq_translations
+
+from requests.exceptions import ConnectionError
+
+logger = logging.getLogger(__name__)
+version = f"v{eq_translations.__version__}"
+
+try:
+    response = requests.get(
+        "https://api.github.com/repos/ONSdigital/eq-translations/releases"
+    )
+    if response.status_code == 200:
+        latest_tag = response.json()[0]["tag_name"]
+        if latest_tag != version:
+            logger.error(
+                'Newer version of eq-translations available, use "pipenv update"'
+            )
+            sys.exit(1)
+    else:
+        logger.error("Can't check eq-translations version")
+        sys.exit(0)
+
+except ConnectionError:
```

Comment:
If we are catching connection errors then I think we should be catching the errors that requests library can raise. All requests exceptions inherit from `requests.exceptions.RequestException` which can include timeouts, invalid response error etc. My point being connection error is far less likely to happen than an actual request error.

Reply:
I agree, we should use `RequestException` rather than `ConnectionError`.

Reply:
Done!👍